build/static/boot/token.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+file_token="$CLUSTER_PATH/cluster_token.txt"
+
+if [[ -n "$DST_TOKEN" ]] && [[ ! -f "$file_token" ]]; then
```

Comment:
Removing `DST` makes it more consistent with the other variables and it's shorter too. And `DST` is redundant anyway, because the token is for running a DST server. Imo it should be `TOKEN`; much better.